src/web/app/actions/beambox/svg-editor.ts

```diff
@@ -2447,16 +2446,9 @@ const svgEditor = (window['svgEditor'] = (function () {
       'beforeunload',
       (e) => {
         if (!isWeb()) return null;
-        // Suppress warning if page is empty
-        if (undoMgr.getUndoStackSize() === 0) {
-          editor.showSaveWarning = false;
-        }
 
-        // showSaveWarning is set to 'false' when the page is saved.
-        if (editor.showSaveWarning) {
-          // Browser already asks question about closing the page
-          e.returnValue = uiStrings.notification.unsavedChanges; // Firefox needs this when beforeunload set by addEventListener (even though message is not used)
-          return uiStrings.notification.unsavedChanges;
```

Comment:
Keep unsaved warning for web

An empty string is enough to trigger the warning.